conference.js

```diff
@@ -2618,24 +2615,14 @@ export default {
      * Toggles the local "raised hand" status.
      */
     maybeToggleRaisedHand() {
```

Comment:
Why this function is "maybe"?

Reply:
I don't even know, actually. This code should go away whenever the new web toolbar gets finished, if that makes you feel any better. I can think of renaming it now.